src/pyhf/tensor/pytorch_backend.py

```diff
@@ -118,6 +127,10 @@ def astensor(self, tensor_in, dtype='float'):
             tensor.shape[0]
         except IndexError:
             tensor = tensor.expand(1)
+
+        if self._mode == "CPU":
```

Comment:
should this be `==GPU`?

Reply:
Oh, I made a mistake. I will modify it in the next commit.